projects/plugins/jetpack/modules/masterbar/admin-menu/class-jetpack-admin-menu.php

```diff
@@ -55,6 +57,21 @@ public function get_preferred_view( $screen, $fallback_global_preference = true
 		return self::DEFAULT_VIEW;
 	}
 
+	/**
+	 * Get the link to the CPT screen. If SSO is enabled, the link will inside calypso,
+	 * otherwise it will be a link to wp-admin.
+	 *
+	 * @param string $post_type The ID of the post type.
+	 * @return string The link to Calypso if SSO is enabled or to WP Admin if SSO is disabled.
+	 */
+	public function get_cpt_menu_link( $post_type ) {
```

Comment:
I thought we maybe needed to support this 'View' switcher too:

<img width="1453" alt="image" src="https://user-images.githubusercontent.com/36432/186271150-70e24232-9adb-4ab1-afac-f76f80b9629f.png">

However, it doesn't seem the feature is supported for Jetpack-connected sites (only Simple and Atomic sites).

Reply:
Correct, this is part of what we call "the unified navbar across Calypso / WP Admin" which is intentionally unsupported on self-hosted Jetpack sites (see https://github.com/Automattic/wp-calypso/issues/66068#issuecomment-1199012300 for more context).